pkg/log/scope.go

```diff
@@ -302,10 +309,18 @@ func (s *Scope) WithLabels(kvlist ...any) *Scope {
 }
 
 func (s *Scope) emit(level zapcore.Level, msg string) {
+	s.emitWithTime(level, msg, time.Now())
+}
+
+func (s *Scope) emitWithTime(level zapcore.Level, msg string, t time.Time) {
```

Comment:
I think this breaks `--log_caller` since now it is nested deeper. Probably we have poor test coverage here (irrespective of your PR)

Reply:
Right, I forgot the caller thing.

Reply:
Oh, the current unit-test capture this error, just because I don't run it. 😂